test-rpc-functions/src/rpc.rs

```diff
@@ -269,6 +280,59 @@ impl RpcTestFunctionsRpcServer for super::RpcTestFunctionsHandle {
 
         Ok(vrf_output.hex_encode())
     }
+
+    async fn generate_transactions(
+        &self,
+        mut input_tx_id: Id<Transaction>,
+        num_transactions: u32,
+        amount_to_spend: u64,
+        fee_per_tx: u64,
+    ) -> rpc::Result<Vec<HexEncoded<SignedTransaction>>> {
+        let coin_decimals = self
+            .call(|this| this.get_chain_config().map(|chain| chain.coin_decimals()))
+            .await
+            .expect("Subsystem call ok")
+            .expect("Subsystem call ok");
```

Comment:
It's a very bad idea to have two expects, right after each other, and with the exact same message... basically if this error happens, it may be difficult to know where it really happened.

Reply:
changed the message